vspd.go

```diff
@@ -74,6 +82,31 @@ func run(ctx context.Context) error {
 	}
 	defer db.Close()
 
+	//Check if adminPass Hash is set in db.
+	hash, err := db.GetAdminHash()
+
+	//Ensure adminpass option is set
+	if cfg.AdminPass == "" && err != nil {
+		log.Error("Config error: the adminpass option is not set")
+		return err
+	}
+
+	if cfg.AdminPass != "" {
+		//Hash the cfg.AdminPass value
+		cfg.AdminPass, err = hashPassword(cfg.AdminPass)
+
+		if err != nil {
+			log.Errorf("Hashing AdminPass Failed: %v", err)
+			return err
+		}
+
+		//if adminpass is set, overwrite the saved adminpass hash in database.
+		db.UpdateAdminPass(cfg.AdminPass)
+
```

Comment:
Empty line space not necessary at the end of a code block.

Reply:
The program should exit after updating the pass; should inform the user that the password has been hashed and saved successfully and ask the user to remove the adminpass option from config and command line before running again.